### DISCUSSION TITLE
Inspirations from nuxt-labs/ui

### DISCUSSION THREAD

**Discussion by AbdallahAlhaddad** [jd-solanki/anu#196] [General]
Hi @jd-solanki,
I found this ui library (https://github.com/nuxtlabs/ui) by the creators of nuxt,  I think we could benefit from thier ideas to improve `anu`, what do you think?

- Could be helpful if we want to support tailwindcss.
- Shortcut feature.
- More ideas to handle type safety.
- better accessibility thanks to `headlessui`.
- More component ideas to add to `anu`.
- ??

**Comment by jd-solanki** [comment]
Thanks for sharing.

- TW: I'm planning to add TailwindCSS support in next release with much more flexibility.
- Shortcuts: I like it, I'll try to implement it
- handle type safety: I'm doing best according to my knowledge if you find is missing feel free to raise the issue. I'll get resolved eventually.
- better accessibility: It's on radar, I'll also handle this in next along with RTL support.
- More component: Components are never ending process, I'll try to add more as soon as I get more time

I constently look for more inspiration to improve Anu always. If you like the lib and want to increase the development efforts, if possible sponsor the project or you can even share some words on twitter 😇 

Thanks pointing this out!

Happy coding 🙌🏻
